recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -1,7 +1,13 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, replace_in_file
+from conan.tools.cmake import CMake
+from conan.errors import ConanInvalidConfiguration
+from conan.errors import ConanException
+
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=2.0"
```

Comment:
For the moment we are also providing compatibility with v1, so depending on the features you use in the recipe this should be a conan 1.X version
```suggestion
required_conan_version = ">=1.53.0"
```